artifactory/commands/commandssummaries/buildinfosummary.go

```diff
@@ -55,3 +85,27 @@ func parseBuildTime(timestamp string) string {
 	// Format the time in a more human-readable format and save it in a variable
 	return buildInfoTime.Format(timeFormat)
 }
+
+func (bis *BuildInfoSummary) generateModuleMarkdown(module buildInfo.Module) string {
```

Comment:
Let's add a unit test for this function.

Reply:
tested as part of `buildInfoModules`

Reply:
OK @RobiNino.